doc/7/controllers/auth/create-api-key/index.md

```diff
@@ -21,20 +21,21 @@ createApiKey(description, [options]);
 
 <br />
 
-| Property | Type | Description |
-| --- | --- | --- |
+| Property      | Type              | Description         |
+| ------------- | ----------------- | ------------------- |
 | `description` | <pre>string</pre> | API key description |
-| `options` | <pre>object</pre> | Additional options |
+| `options`     | <pre>object</pre> | Additional options  |
 
 ### options
 
 Additional query options
 
-| Property | Type<br />(default) | Description |
-| --- | --- | --- |
-| `expiresIn` | <pre>string/number</pre><br />(`-1`) | Expiration duration |
-| `_id` | <pre>string</pre><br />(`null`) | API key unique ID |
-| `refresh` | <pre>boolean</pre><br />(`false`) | If set to `wait_for`, Kuzzle will not respond until the API key is indexed |
+| Property    | Type<br />(default)                  | Description                                                                                                           |
+| ----------- | ------------------------------------ | --------------------------------------------------------------------------------------------------------------------- |
+| `expiresIn` | <pre>string/number</pre><br />(`-1`) | Expiration duration                                                                                                   |
+| `_id`       | <pre>string</pre><br />(`null`)      | API key unique ID                                                                                                     |
+| `refresh`   | <pre>boolean</pre><br />(`false`)    | If set to `wait_for`, Kuzzle will not respond until the API key is indexed                                            |
+| `timeout`   | <pre>number</pre><br/>(`-1`)         | Time (in ms) during which a request will still be waited to be resolved. Set it `-1` if you want to wait indefinitely |
```

Comment:
`Set it -1 if you want to wait indefinitely`
Well, yeah but not really... -1 is the default right?

Reply:
There is a global default which can be configured with the `options` of the Kuzzle Constructor but is set to `-1` by default
Then each method can override this configuration on a per request basis.